react/features/conference/components/native/constants.js

```diff
@@ -0,0 +1,125 @@
+import { TransitionPresets } from '@react-navigation/stack';
+import React from 'react';
+import { Platform, TouchableWithoutFeedback } from 'react-native';
+
+import { Icon, IconArrowBack } from '../../../base/icons';
+
+import * as ConferenceNavigationContainerRef
+    from './ConferenceNavigationContainerRef';
+
+
+/**
+ * Default modal transition for the current platform.
+ */
+export const conferenceModalPresentation = Platform.select({
+    ios: TransitionPresets.ModalPresentationIOS,
+    default: TransitionPresets.DefaultTransition
+});
+
+/**
+ * Screen options and transition types.
+ */
+export const screenOptions = {
+    ...TransitionPresets.ModalTransition,
+    gestureEnabled: false,
+    headerShown: false
+};
+
+/**
+ * Screen options for conference.
+ */
+export const conferenceScreenOptions = {
+    ...screenOptions
+};
+
+/**
+ * Screen options for lobby modal.
+ */
+export const lobbyScreenOptions = {
+    ...screenOptions
+};
+
+/**
+ * Tab bar options for chat screen.
+ */
+export const chatTabBarOptions = {
+    activeTintColor: '#17a0db',
+    labelStyle: {
+        fontSize: 12
+    },
+    inactiveTintColor: '#666',
+    indicatorStyle: {
+        backgroundColor: '#17a0db'
+    }
+};
+
+/**
+ * Screen options for chat.
+ */
+export const chatScreenOptions = {
+    ...TransitionPresets.DefaultTransition,
+    headerBackTitleVisible: false,
+    headerLeft: () => (
+        // eslint-disable-next-line react/react-in-jsx-scope
+        <TouchableWithoutFeedback
+            /* eslint-disable-next-line react/jsx-no-bind */
+            onPress = { () => ConferenceNavigationContainerRef.navigate('Conference') } >
+            {/* eslint-disable-next-line react/react-in-jsx-scope */}
+            <Icon
+                size = { 20 }
+                src = { IconArrowBack }
+                /* eslint-disable-next-line react-native/no-inline-styles */
+                style = {{ marginLeft: 12 }} />
+        </TouchableWithoutFeedback>
+    ),
+    headerShown: true,
+    headerStyle: {
+        backgroundColor: '#17A0DB'
+    },
+    headerTitleStyle: {
+        color: '#fff'
+    },
+    headerTintColor: '#fff'
```

Comment:
Can you check if this is necessary in the end?

Reply:
Removed